doc/module-developer/src/output/output-variable-dos-and-don-ts.tex

```diff
@@ -31,7 +31,7 @@ \subsection{Output Variable Naming Conventions}\label{output-variable-naming-con
   Output variable names shall always be less than 100 characters long.
 \end{itemize}
 
-An output variable name should be constructed from up to six separate name elements.~ The following diagram shows the elements and the order that they are to be used in the name.~ The first three are each somewhat optional and are used as needed to identify the type of engineering model, object, or device being reported on.~ The last three are nearly always needed. Units are always needed (as described below.
+An output variable name should be constructed from up to five separate name elements.~ The following diagram shows the elements and the order that they are to be used in the name.~ The first three are each somewhat optional and are used as needed to identify the type of engineering model, object, or device being reported on.~ The last two are nearly always needed. .
```

Comment:
Stray period at the end of this paragraph; fix it if more commits are needed, not worth fixing if there's no other commits.